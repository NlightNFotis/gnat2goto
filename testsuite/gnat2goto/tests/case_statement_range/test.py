from test_support import *

prove(main='main.adb')
