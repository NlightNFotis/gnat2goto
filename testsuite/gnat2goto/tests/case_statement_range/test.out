[case_statement_range.assertion.1] line 6 assertion false: SUCCESS
[case_statement_range.assertion.2] line 8 assertion false: SUCCESS
[main.assertion.1] line 5 assertion Case_Statement_Range = 20: SUCCESS
VERIFICATION SUCCESSFUL
